internal/resources/metal/organization_member/resource.go

```diff
@@ -0,0 +1,281 @@
+			return
+		}
+		return
+	}
+
+	members, _, err := client.Members.List(orgID, &packngo.GetOptions{Includes: []string{"user"}})
+	if err != nil {
+		err = equinix_errors.FriendlyError(err)
+		// If the org was destroyed, mark as gone.
+		if equinix_errors.IsNotFound(err) {
+			data.OrganizationID = basetypes.NewStringNull()
+			return
+		}
+		return
+	}
+	member, err := findMember(invitee, members, invitations)
+	if err != nil {
+		log.Printf("[WARN] Could not find member %s in organization, removing from state", data.OrganizationID)
+		data.OrganizationID = basetypes.NewStringNull()
+		return
+	}
+
+	if member.isMember() {
+		projectsList, diags := types.SetValueFrom(context.Background(), types.StringType, member.Member.Projects)
+		if diags.HasError() {
+			return
+		}
+		data.ProjectIDs = projectsList
+		data.State = types.StringValue("active")
+
+		rolesList, diags := types.SetValueFrom(context.Background(), types.StringType, member.Member.Roles)
+		if diags.HasError() {
+			return
+		}
+		data.Roles = rolesList
+		data.OrganizationID = types.StringValue(member.Member.Organization.URL)
+
+		// data.Created = types.StringValue(member.CreatedAt.String())
+		// data.Updated = types.StringValue(member.UpdatedAt.String())
+	} else if member.isInvitation() {
+		projectsList, diags := types.SetValueFrom(context.Background(), types.StringType, member.Member.Projects)
+		if diags.HasError() {
+			return
+		}
+		data.ProjectIDs = projectsList
+		data.State = types.StringValue("active")
+
+		rolesList, diags := types.SetValueFrom(context.Background(), types.StringType, member.Member.Roles)
+		if diags.HasError() {
+			return
+		}
+		data.Roles = rolesList
+		data.OrganizationID = types.StringValue(member.Member.Organization.URL)
+		data.Created = types.StringValue(member.Invitation.CreatedAt.String())
+		// data.Updated = types.StringValue(member.Invitation.UpdatedAt.String())
+		data.Nonce = types.StringValue(member.Invitation.Nonce)
+		data.Invited_by = types.StringValue(path.Base(member.Invitation.InvitedBy.Href))
+	}
+
+	// Set state to fully populated data
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func findMember(invitee string, members []packngo.Member, invitations []packngo.Invitation) (*member, error) {
+	for _, mbr := range members {
+		if mbr.User.Email == invitee {
+			return &member{Member: &mbr}, nil
+		}
+	}
+
+	for _, inv := range invitations {
+		if inv.Invitee == invitee {
+			return &member{Invitation: &inv}, nil
+		}
+	}
+	return nil, fmt.Errorf("member not found")
+}
+
+func (r *Resource) Delete(
+	ctx context.Context,
+	req resource.DeleteRequest,
+	resp *resource.DeleteResponse,
+) {
+	tflog.Debug(ctx, "Delete Organization")
+	r.Meta.AddFwModuleToMetalUserAgent(ctx, req.ProviderMeta)
+	client := r.Meta.Metal
+
+	var data ResourceModel
+
+	listOpts := &packngo.ListOptions{Includes: []string{"user"}}
+	invitations, _, err := client.Invitations.List(data.OrganizationID.String(), listOpts)
```

Comment:
I think all of the `.String()` calls in this file are supposed to be `.ValueString()`

Per the note on this page, the `.String()` function is only intended for debugging: https://developer.hashicorp.com/terraform/plugin/framework/handling-data/types/string#accessing-values